modules/node_modules/@ncigdc/components/Modals/NoAccessModal.js

```diff
@@ -4,35 +4,46 @@ import { connect } from 'react-redux';
 
 import { setModal } from '@ncigdc/dux/modal';
 import Button from '@ncigdc/uikit/Button';
+import { Column, Row } from '@ncigdc/uikit/Flex';
 import LoginButton from '../LoginButton';
 
 const LoginModal = ({
   dispatch,
   message = "You don't have access",
+  primaryButton,
 }: {
   dispatch: Function,
   message: string,
 }) => (
-  <span>
-    <h2 style={{ paddingLeft: 15 }} >Access Error</h2>
-    <div
+  <Column
+    style={{ margin: '-15px' }}
```

Comment:
I think if we want those lines to go all the way across it would be better to remove the padding from the modalContainer. Each modal can add their padding then.